### PR TITLE
chore: Towards the next nightly

### PR DESCRIPTION
**Pull request by Xuanwo**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Rust 1.61 has been released, let's go towards the next nightly.

## Changelog

- New Feature

### CODE REVIEW

**Comment by Xuanwo** [comment]
@everpcpc Can you help build the `datafuselabs/build-tool:dev-nightly-2022-05-19` image? Thanks in advance.

**Comment by Xuanwo** [comment]
@mergify update

**Comment by Xuanwo** [comment]
Oh, I got it. We need to wait the docker build finished on `main` branch.

**Comment by Xuanwo** [comment]
@mergify update

**Review by everpcpc — APPROVED** [review]

**Comment by BohuTANG** [comment]
Failed at:
```
---- scalars::others::inet_aton::test_inet_aton_function stdout ----
thread 'scalars::others::inet_aton::test_inet_aton_function' panicked at 'assertion failed: `(left == right)`: invalid input

Diff < left / right > :
<Failed to parse 'batman' into a IPV4 address, invalid IP address syntax
>Failed to parse 'batman' into a IPV4 address, invalid IPv4 address syntax
```

**Comment by Xuanwo** [comment]
> Failed at:
> 
> ```
> ---- scalars::others::inet_aton::test_inet_aton_function stdout ----
> thread 'scalars::others::inet_aton::test_inet_aton_function' panicked at 'assertion failed: `(left == right)`: invalid input
> 
> Diff < left / right > :
> <Failed to parse 'batman' into a IPV4 address, invalid IP address syntax
> >Failed to parse 'batman' into a IPV4 address, invalid IPv4 address syntax
> ```

Should be addressed.

**Review by PsiACE — APPROVED** [review]

File: metasrv/src/meta_service/raftmeta.rs [hunk 1]
```diff
@@ -554,13 +554,12 @@ impl MetaNode {
...
 
-        let rst = self
-            .raft
+        self.raft
             .initialize(cluster_node_ids)
             .await
             .map_err(|x| MetaError::MetaServiceError(format!("{:?}", x)))?;
 
-        tracing::info!("initialized cluster, rst: {:?}", rst);
+        tracing::info!("initialized cluster");
```
Comment (drmingdrmer) [inline metasrv/src/meta_service/raftmeta.rs:562]: 👍

**Pull request merged**